.gitattributes

```diff
@@ -0,0 +1,4 @@
+docker/ export-ignore
+.clang* export-ignore
+.ycm_extra_conf.py export-ignore
+.travis.yml export-ignore
```

Comment:
Add .mailmap

Reply:
Do we still need `.project` and `.cproject`?

Reply:
They are unnecessary